src/js/component.js

```diff
@@ -1023,6 +1041,7 @@ class Component {
     const report = Lib.bind(this.player(), this.player().reportUserActivity);
 
     let touchHolding;
```

Comment:
Yeah, we should. Unfortunately (fortunately?), eslint doesn't have a rule about it for lets.